examples/stable_memory/src/stable_memory.ts

```diff
@@ -0,0 +1,9 @@
+import { ic, nat32, nat64, Query, Update } from 'azle';
+
+export function stable_size(): Update<nat32> {
```

Comment:
stable_size should also be a query function

Reply:
This should get resolved with this PR
https://github.com/demergent-labs/azle/pull/465#pullrequestreview-1026719492